scripts/xtensa-build-zephyr.py

```diff
@@ -902,6 +908,73 @@ def build_platforms():
 			symlinks=True, ignore_dangling_symlinks=True, dirs_exist_ok=True)
 
 
+def install_lib(platform, sof_output_dir, abs_build_dir):
+	"""[summary] Sign loadable llext modules, if any, copy them to the
+	deployment tree and create UUID links for the kernel to find and load
+	them."""
+
+	global signing_key
+	global platform_wconfig
+
+	with os.scandir(str(abs_build_dir)) as iter:
+		rimage_executable = platform_wconfig.get("rimage.path")
+		if rimage_executable is None:
+			rimage_executable = platform_wconfig.get("rimage.path", None,
+								 west_config.ConfigFile.LOCAL)
+
+		for entry in iter:
+			if (not entry.is_dir or
+			    not entry.name.endswith('_llext')):
+				continue
+			uuids = pathlib.Path(entry.path) / 'llext.uuid'
+			if not os.path.exists(uuids):
+				continue
+
+			# replace '_llext' with '.llext', e.g.
+			# eq_iir_llext/eq_iir.llext
+			llext_base = entry.name[:-6]
+			llext_file = llext_base + '.llext'
+
+			sof_lib_dir = sof_output_dir / '..' / 'sof-ipc4-lib' / platform
+			if args.key_type_subdir != "none":
+				sof_lib_dir = sof_lib_dir / args.key_type_subdir
+			sof_lib_dir.mkdir(parents=True, exist_ok=True)
+			dst = sof_lib_dir / llext_file
+
+			rimage_cfg = pathlib.Path(entry.path) / 'rimage_config.toml'
+			llext_input = pathlib.Path(entry.path) / str(llext_base + '.so')
+			llext_output = pathlib.Path(entry.path) / llext_file
+
+			sign_cmd = [str(rimage_executable), "-o", str(llext_output),
+				    "-e", "-c", str(rimage_cfg),
+				    "-k", str(signing_key), "-l", "-r",
+				    str(llext_input)]
+			execute_command(sign_cmd, cwd=west_top)
+
+			# Explicitly rely on default exception handling
+			fdst = open(str(dst), 'wb')
```

Comment:
`open()` does not need `str()`.
```suggestion
			fdst = open(dst, 'wb')
```